vimiv/gui/metadatawidget.py

```diff
@@ -13,27 +13,29 @@
 from PyQt5.QtWidgets import QLabel, QSizePolicy, QWidget
 
 from vimiv import api, utils
-from vimiv.imutils import exif
+from vimiv.imutils import metadata
 from vimiv.config import styles
 
 _logger = utils.log.module_logger(__name__)
 
 
-if exif.has_exif_support:
+# TODO: conditional fails as this file is imported before plugins are loaded
+# if metadata.has_get_metadata():
+if True:
```

Comment:
Adding comment as a reminder that this is still on the todo list :blush:

Reply:
I pushed an idea to `core/metadataplugin-jcjgraf`, what do you think about solving this with a `plugins_loaded` signal? I have the feeling this could be useful in general :thinking: 

But definitely doesn't help with `--version` ...

Reply:
I pushed an idea to `core/metadataplugin-jcjgraf`, what do you think about solving this with a `plugins_loaded` signal? I have the feeling this could be useful in general :thinking: 

But definitely doesn't help with `--version` ...

Reply:
Is currently used, may change later in case of internal keys, but we can stick to this for now IMHO. Feel free to re-open in case you think differently.

Reply:
Updated PR Description/todo that we remember this